packages/studio-plugin/src/createStudioStylingPlugin.ts

```diff
@@ -46,7 +46,10 @@ function isImportedByStudio(importer: string) {
 
 function isStyleFile(id: string) {
   const idWithoutQueryParams = id.split("?")[0];
-  return !!idWithoutQueryParams.endsWith(".css");
+  return !!(
```

Comment:
nit: can the double negation be removed now?

Reply:
done!